integration_tests/tests/billing-and-transaction-view-tests.ts

```diff
@@ -37,11 +38,17 @@ describe("\nUpload pdf invoice to raw invoice bucket and verify BillingAndTransa
   `(
     "results retrieved from billing and transaction_curated view query should match with expected $testCase,$billingQty,$billingPriceFormatted,$transactionQty,$transactionPriceFormatted,$priceDifferencePercentage",
     async ({ ...data }) => {
-      await generateTransactionEventsViaFilterLambda(
-        data.eventTime,
-        data.transactionQty,
-        data.eventName
-      );
+      for (let i = 0; i < data.transactionQty; i++) {
+        const createEventPayload = await generateTestEvent({
```

Comment:
Because `generateTestEvent` returns an event payload, I think the variable name can just be eventPayload. 
```suggestion
        const eventPayload = await generateTestEvent({
```

Reply:
done